support/cas-server-support-fortress/build.gradle

```diff
@@ -0,0 +1,21 @@
+description = "Apereo CAS Fortress Support"
+dependencies {
+    compile project(":core:cas-server-core-util")
+    compile project(":core:cas-server-core-authentication")
+    compile project(":core:cas-server-core-configuration")
+    compile ("org.apache.directory.fortress:fortress-core:1.0.1") {
```

Comment:
Please move this block into the dependencies.gradle file, and reference is as libraries.fortress.